contracts/contribute/Contributors.sol

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+/// @dev Only `owner` has a privilege, but the `sender` was provided.
+/// @param sender Sender address.
+/// @param owner Required sender address as an owner.
+error OwnerOnly(address sender, address owner);
+
+/// @dev The contract is already initialized.
+error AlreadyInitialized();
+
+/// @dev Zero address.
+error ZeroAddress();
+
+/// @dev Only manager is allowed to have access.
+error OnlyManager(address sender, address manager);
+
+/// @dev Wrong length of two arrays.
+/// @param numValues1 Number of values in a first array.
+/// @param numValues2 Number of values in a second array.
+error WrongArrayLength(uint256 numValues1, uint256 numValues2);
+
+/// @dev Account is unauthorized.
+/// @param account Account address.
+error UnauthorizedAccount(address account);
+
+/// @title Contributors - Smart contract for managing contributors
+/// @author Aleksandr Kuperman - <aleksandr.kuperman@valory.xyz>
+/// @author Andrey Lebedev - <andrey.lebedev@valory.xyz>
+/// @author Tatiana Priemova - <tatiana.priemova@valory.xyz>
+/// @author David Vilela - <david.vilelafreire@valory.xyz>
+contract Contributors {
```

Comment:
Storage contract for contributors. For now just a basic functionality, as it's proxy-implementation based.